kernel/kernel-api/src/main/java/io/delta/kernel/client/ExpressionHandler.java

```diff
@@ -0,0 +1,25 @@
+package io.delta.kernel.client;
+
+import io.delta.kernel.expressions.Expression;
+import io.delta.kernel.expressions.ExpressionEvaluator;
+import io.delta.kernel.types.StructType;
+
+/**
+ * Provides expression evaluation capability to Delta Kernel. Delta Kernel can use this client
+ * to evaluate predicate on partition filters, fill up partition column values and any computation
+ * on data using {@link Expression}s.
+ */
+public interface ExpressionHandler
+{
+    // TODO: specify 1:1 input and outout
```

Comment:
Remove this TODO and add the comment to the javadoc.

Reply:
What does 1:1 input and output mean here?

Reply:
It means,  the output contains one row for each row of the input.

Reply:
So this is for ExpressionEvaluator.eval essentially? It's already specified there. I'll remove this TODO